test/kata3.numberToReversedDigits.test.js

```diff
@@ -1,5 +1,7 @@
 const { numberToReversedDigits } = require("../src");
 
 describe("numberToReversedDigits", () => {
-  test("returns a reversed array of the number's digits", () => {});
+  test("returns a reversed array of the number's digits", () => {
```

Comment:
Could any more tests be added here to make this test a bit more valuable? Things like numbers of different length would be good to check